packages/cdktf/lib/testing/__tests__/matchers.test.ts

```diff
@@ -239,6 +239,11 @@ describe("matchers", () => {
           "Expected subject to be a valid terraform stack"
         )
       );
+      expect(res.message).toEqual(
+        expect.stringContaining(
```

Comment:
Would be nice to check/use more of the output here.

Reply:
@Maed223 Added another test case a few lines down below 👍